products/centreon.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Centreon
+category: server-app
+tags: php-runtime
+permalink: /centreon
+versionCommand: cat /etc/centreon/centreon.conf
+releasePolicyLink: https://docs.centreon.com/docs/releases/lifecycle/
+releaseDateColumn: true
+activeSupportColumn: true
+eolColumn: Security Support
+
+#identifiers:
+#-   repology: zabbix
+#-   purl: pkg:brew/zabbix
+#-   purl: pkg:github/zabbix/zabbix
+
+auto:
+-   git: https://github.com/centreon/centreon.git
```

Comment:
There are lots of different tags on https://github.com/centreon/centreon/tags. This misses a regex to filter those, but I don't know which one to choose (maybe `centreon-web` ?). Or maybe we should not document latest versions at all ?

Reply:
Yes, that's a keypoint 😓 
I don't know which one should be the target. I suppose it is the web one, but unfortunately I cannot say better.
👉 Should I ask them through a dedicated issue ?

Reply:
- https://github.com/centreon/centreon/issues/2237